scripts/autoversion

```diff
@@ -2,12 +2,10 @@
 set -euo pipefail
 
 old_version="$(svu current)"
-new_version="$(svu next)"
+new_version="$(svu next --force-patch-increment)"
```

Comment:
I had that in my initial implementation, until I realized the behaviour is the same. `svu next --force-patch-increment` will give the same output as `svu next`, unless old == new in which case it will increment the micro.

Reply:
Aaah, nice 👍